frontend/lib/controllers/controller_constant.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:frontend/controllers/api_controller.dart';
+import 'package:frontend/controllers/task_controller.dart';
+
+ApiController apiController = ApiController.instance;
+TaskController taskController = TaskController.instance;
```

Comment:
Why not prefixed by `const` is it's constant?
Constant should be write in `SCREAM_CASE`